lecture_5/homework/spec/requests/buildings_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+require 'pry'
+
+RSpec.describe 'Buildings API', type: :request do
+
+  let!(:strongholds) { create_list(:stronghold, 5) }
+  let(:stronghold_long_survive) { create(:stronghold, granary: 100) }
+
+  describe 'GET /builidings' do
+
+    before { get '/buildings' }
+
+    it 'returns all buidlings' do
+      expect(json).not_to be_empty
+      expect(json["data"].size).to eq(5)
```

Comment:
Za dobrą praktykę przyjęło się, że w jednym it nie tworzy się paru expect :) Wygodniej się takie testy czyta i formatuje ich output